src/Compilers/CSharp/Portable/Symbols/Source/SourceModuleSymbol.cs

```diff
@@ -619,6 +619,39 @@ internal override void AddSynthesizedAttributes(ModuleCompilationState compilati
             }
         }
 
+        private NamedTypeSymbol _lazyNullableAttribute;
+
+        internal NamedTypeSymbol GetNullableAttribute()
```

Comment:
>GetNullableAttribute [](start = 33, length = 20)

Is this a temporary helper? Consider adding a PROTOTYPE comment.

Reply:
Added comment.